manim/mobject/svg/text_mobject.py

```diff
@@ -193,16 +192,15 @@ def gen_chars(self, lines_str_list):
         """
         char_index_counter = 0
         chars = self.get_group_class()()
-        for line_no in range(lines_str_list.__len__()):
+        for line_no in range(len(lines_str_list)):
             chars.add(self.get_group_class()())
             chars[line_no].add(
                 *self.lines_text.chars[
                     char_index_counter : char_index_counter
-                    + lines_str_list[line_no].__len__()
-                    + 1
+                    + len(Text(lines_str_list[line_no]))
                 ]
             )
-            char_index_counter += lines_str_list[line_no].__len__() + 1
+            char_index_counter += len(Text(lines_str_list[line_no]))
```

Comment:
Seems this is only created to get the length of the Text right? Is there a way to avoid creating this? It seems to impact performance for the opengl renderer (that already has performance issues)